src/tox/session/__init__.py

```diff
@@ -220,6 +221,28 @@ def subcommand_test(self):
             retcode = self._summary()
         return retcode
 
+    def _add_parallel_summaries(self):
+        if self.config.option.parallel == PARALLEL_OFF:
```

Comment:
``src/tox/session/__init__.py (82.8%): Missing lines 228,232,238,240,242``

Maybe just put things nested and will be more readable and not have to create artificial tests for these.

Reply:
Also, our changelog user format is by `` :user:`zeroshift` ``

Reply:
Then https://github.com/tox-dev/tox/blob/master/.github/PULL_REQUEST_TEMPLATE.md should be updated.

Reply:
As far as I can tell you already fixed the coverage etc.

Reply:
Indeed, I've updated the template thanks 👍 for catching that. I think the PR is ready just waiting for the PR to pass and will trigger a tox release after.